src/com/trollworks/gcs/advantage/AdvantageColumn.java

```diff
@@ -72,19 +72,31 @@ public Object getData(Advantage advantage) {
         @Override
         public String getDataAsText(Advantage advantage) {
             StringBuilder builder = new StringBuilder();
-            String        notes   = advantage.getModifierNotes();
-
-            builder.append(advantage.toString());
```

Comment:
And error on my part.   I did not understand the complete usage of getDataAsText().   I have added that line back in (so the row sorter works correctly).   Also, this method uses the "*InDisplay" preferences, since this data (and the row sorter) are using during the main display.